Sources/Controllers/2.0/OADeleteCustomFiltersViewController.mm

```diff
@@ -125,14 +125,13 @@ - (void)setupDeleteButtonView
 
 - (nonnull UITableViewCell *)tableView:(nonnull UITableView *)tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath
 {
-    NSObject *item = _items[indexPath.row];
     NSString *cellType = indexPath.row == 0 ? [OACustomSelectionButtonCell getCellIdentifier] : [OAMenuSimpleCell getCellIdentifier];
     if ([cellType isEqualToString:[OACustomSelectionButtonCell getCellIdentifier]])
     {
-        OACustomSelectionButtonCell *cell = [tableView dequeueReusableCellWithIdentifier:[OACustomSelectionButtonCell getCellIdentifier]];
```

Comment:
Please do not regress the code to the incorrect state. Always prefer code from master when fixing the conflicts.